pkgs/misc/qtscrcpy/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, libsForQt5
+, copyDesktopItems
+, android-tools
+, makeDesktopItem
+, lib
+}:
```

Comment:
```suggestion
{ stdenv
, fetchFromGitHub
, makeWrapper
, wrapQtAppsHook
, cmake
, pkg-config
, qttools
, qtx11extras
, ffmpeg
, copyDesktopItems
, android-tools
, makeDesktopItem
, lib
}:
```

Reply:
error: anonymous function at default.nix:1:1 called without required argument 'wrapQtAppsHook'
I'm using `nix-build -E 'with import <nixpkgs> { }; callPackage ./default.nix { }' -v` command to build

Reply:
Giving an FFmpeg fixes the build on aarch64, good!